docs/gettingstarted/query_test_report.rst

```diff
@@ -63,6 +63,17 @@ To print just the raw format fields use the command ``buildtest report --formatf
 
     .. command-output:: buildtest report --formatfields
 
+Format Reports (``buildtest report --detailed``)
+-----------------------------------------------
```

Comment:
the underline needs to be till end of the title it looks like there needs to be one more `-` or few more

Reply:
Done!